src/unicef_attachments/fields.py

```diff
@@ -81,3 +81,37 @@ def __call__(self, serializer_field):
 
     def __repr__(self):
         return "%s()" % self.__class__.__name__
+
+
+class AbsoluteUrlField(serializers.CharField):
+    """
+    build absolute url automatically using current domain when possible
+    """
+
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault("read_only", True)
+        super().__init__(*args, **kwargs)
+
+    def to_representation(self, value):
+        if not value:
+            return value
+
+        request = self.context.get("request", None)
+        if request is None:
+            return value
+
+        return request.build_absolute_uri(value)
```

Comment:
we're using blob storage, it's got some of its own magic. do we need this?

Reply:
this logic doesn't depend on storage. here we replace relative backend url with absolute one
Attachment.file_link generates url which allows us to download attachment with respect of permissions, but it's not absolute